webpack.build.js

```diff
@@ -0,0 +1,57 @@
+const path = require("path");
+
+module.exports = {
+  entry: {
+    main: "./src/index.tsx",
+  },
+  output: {
+    filename: "main.js",
+    path: path.resolve(__dirname, "dist"),
+    libraryTarget: "commonjs",
+  },
+  externals: {
+    // Don't bundle react or react-dom
```

Comment:
Can you explain why?

Reply:
If we bundle react and react-dom, and the app/page that imports our component is using a different version it blows up.

If anyone wanted to use it "standalone" then they could copy the example, but in general (and for us) it will be part of a larger app that will have it's own react